README

```diff
@@ -177,6 +177,8 @@ Compiler Notes
   source directory path names that was resolved in 9.0-4 (9.0-3 is
   known to be broken in this regard).
 
+- The PGI compiler is not supported on OS-X.
```

Comment:
Nit picks:

* The PGI compiler itself may be supported on OS X -- we don't want to say that it isn't.  We want to say that Open MPI doesn't support building with the PGI compiler on OS X.
* It's not "OS-X".  It's "OS X" and "MacOS".

How about:

> Open MPI does not support the PGI compiler suite on OS X or MacOS.